<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar {
border: 0.1px solid grey;
background-color: #50535C;
}

QProgressBar::chunk {
background-color: #00C258;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>864</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/bg3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>395</width>
      <height>106</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/warning.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_zhu_zhou">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>268</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fu_zhou">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>268</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>974</x>
      <y>268</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1_gang">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>800</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2_gang">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>800</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3_gang">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>800</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="prgBar_1_chang">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>358</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="prgBar_2_chang">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>445</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="prgBar_3_chang">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>533</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="prgBar_4_chang">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>621</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="prgBar_5_chang">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>711</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="prgBar_ji_xie_bi">
    <property name="geometry">
     <rect>
      <x>967</x>
      <y>380</y>
      <width>208</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>500</number>
    </property>
    <property name="maximum">
     <number>1200</number>
    </property>
    <property name="value">
     <number>800</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_wen_du">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>268</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
font: 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>0000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>700</y>
      <width>125</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSub">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>760</y>
      <width>125</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>958</x>
      <y>479</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(0,0,0,0);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_over_layer">
    <property name="geometry">
     <rect>
      <x>981</x>
      <y>502</y>
      <width>148</width>
      <height>105</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/overlay.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>主机温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>主轴转速</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>副轴转速</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>主机时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>326</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>1号仓压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>415</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>2号仓压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>505</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>3号仓压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>594</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>4号仓压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>682</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>5号仓压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>770</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>1号工作缸油量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>771</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>2号工作缸油量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>681</x>
      <y>771</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>3号工作缸油量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>353</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>机械臂压力</string>
    </property>
   </widget>
   <widget class="QLabel" name="chang1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>326</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="chang2">
    <property name="geometry">
     <rect>
      <x>137</x>
      <y>415</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="chang3">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>506</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="chang4">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>595</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="chang5">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>682</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_ji_xie_bi">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>351</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;新宋体&quot;;
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>0000 Pa</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
